src/api/dependency-discovery/README.md

```diff
@@ -28,7 +28,17 @@ By default, the server is listening on port 10500 (http://localhost:10500).
 
 ### Docker
 
-TODO
+To run the service with only the container, you may use the following command:
+
+```bash
+docker run -p 10500:10500 telescope_dependency_discovery_svc:latest
+```
+
+You would have to build the container prior:
+
+```bash
+docker build . -t telescope_dependency_discovery_svc:latest
```

Comment:
I'm curious, `telescope_dependency_discovery_svc` is the tag name and `latest` is the tag version right?

Reply:
Correct! The tag name will always be the same, while the version may change depending on the version you want to support (`latest`, `stable`, `v1.4`, etc). Because we are not uploading them to a registry (yet), the tag version is not that relevant.